### PR TITLE
Preserve empty lines in <pre></pre> blocks

### PR DESCRIPTION
**Pull request by kpagacz**
Let me know if you want to change the names of the functions or do something else entirely.

### CODE REVIEW

File: src/render/text_renderer.rs [hunk 1]
```diff
@@ -329,13 +329,17 @@ impl<T: Clone + Eq + Debug + Default> WrappedBlock<T> {
...
         if !self.line.is_empty() {
-            let mut tmp_line = TaggedLine::new();
-            mem::swap(&mut tmp_line, &mut self.line);
-            self.text.push(tmp_line);
-            self.linelen = 0;
+            self.flush_preformatted_line();
         }
     }
 
+    fn flush_preformatted_line(&mut self) {
```
[the remark this answers is not in the document]
Comment (kpagacz) [reply src/render/text_renderer.rs]: Changed to `force_flush_line`

**Comment by jugglerchris** [comment]
Perfect, thank you!
Do you need a new release with this fix in the relatively near future?

**Comment by kpagacz** [comment]
My workaround is good enough for now. If it is not too much of a hassle, I would be happy with a release, but I am also good to be on the lookout for the new version in a couple of months.

**Comment by jugglerchris** [comment]
I've just released 0.4.5.  Let me know if there are any further problems!

**Pull request merged**